### PR TITLE
feature(#6) Implemented close task subscription and gracefully shutting dow…

### PR DESCRIPTION
**Pull request by jsam**
…n the connection

### CODE REVIEW

File: examples/task-svc/main.go [hunk 1]
```diff
@@ -61,7 +61,14 @@ func openSubscription(client *zbc.Client, stopCh chan bool, pid int32, topic str
 			break
 
 		case stop := <-stopCh:
-			if stop {
+			if stop == true {
```
Comment (menski) [inline examples/task-svc/main.go]: is that `== true` needed?

File: examples/task-svc/main.go [hunk 2]
```diff
@@ -71,7 +78,7 @@ func openSubscription(client *zbc.Client, stopCh chan bool, pid int32, topic str
 
 func startWorkerView(w http.ResponseWriter, r *http.Request) {
 	resp := make(map[string]interface{})
-	zbClient, err := zbc.NewClient("0.0.0.0:51015")
+	zbClient, err := zbc.NewClient("192.168.21.72:51015")
```
Comment (menski) [inline examples/task-svc/main.go]: I think it would be good to not commit the container IP address

File: zbc/client.go [hunk 3]
```diff
@@ -108,16 +112,24 @@ func (c *Client) TaskConsumer(ts *TaskSubscription) (chan *Message, error) {
 		log.Println(err)
 		return nil, err
 	}
-	c.subscriptions[(*response.Data)["subscriberKey"].(uint64)] = subscriptionCh
+
+	subscriberKey := (*response.Data)["subscriberKey"].(uint64)
+	log.Println(subscriberKey)
```
Comment (menski) [inline zbc/client.go]: do you want to keep this debug output?

File: examples/task-svc/main.go (hunk 1 above)
Comment (jsam) [inline examples/task-svc/main.go]: gone.

File: examples/task-svc/main.go (hunk 2 above)
Comment (jsam) [inline examples/task-svc/main.go]: :+1:

File: zbc/client.go (hunk 3 above)
Comment (jsam) [inline zbc/client.go]: gone

**Pull request closed without merging**